docs/content/docs/development_guide.md

```diff
@@ -59,6 +59,7 @@ as a birds-eye view of where the different components are located.
     - `smi/` - SMI client, informer, caches and tools
     - `tests/` - test fixtures and other functions to make unit testing easier
     - `trafficpolicy/` - SMI related types
+  - `wasm/` - Source for a WebAssembly-based Envoy extension
```

Comment:
indentation seems off

Reply:
This looks right to me since `wasm` is at the rot of the repo, not under `pkg/`.